lib/core/io.js

```diff
@@ -298,7 +297,7 @@ class XHRReader extends Reader {
 				} else {
 					resolve();
 				}
-			}, reject));
+			}, reject, {range: "bytes=0,0"}));
```

Comment:
`{ range: "bytes=0-0"}` must be replaced with `[["range", "bytes=0-0"]]`

Reply:
Done.